modules/manage/pages/use-admin-api.adoc

```diff
@@ -0,0 +1,124 @@
+
+== Prerequisites
+
+* A running Redpanda Self-Managed cluster.
+* Superuser privileges, if xref:manage:security/authentication.adoc#enable-authentication[authentication] is enabled on your cluster for the Admin API. For more information, see xref:manage:security/authentication.adoc#create-superusers[Configure Authentication]. (Some endpoints are read-only and do not require superuser access.)
+* A tool to make HTTP requests, such as `curl`, or client libraries for your programming language of choice. 
+** For Admin API operations introduced in v25.3 and later, you can also make requests using a ConnectRPC client. You can install the Connect plugin for your preferred language and use the Protobuf compiler to generate an SDK. These RPC services are also available in a Buf module and can be accessed through the https://buf.build/redpandadata/core/docs/dev:redpanda.core.admin.v2[Buf Schema Registry]. The https://buf.build/docs/cli/[Buf CLI] provides an easy way to generate client SDKs.
+
+== Use the Admin API
+
+Starting in Redpanda v25.3, the Admin API serves new endpoints as ConnectRPC services, which allows you to continue to invoke them using HTTP requests. Both new and legacy (RESTful) endpoints are accessible on the same port (default: 9644), but they use different URL paths.
+
+NOTE: Legacy Admin API endpoints remain available and fully supported. Use them for operations not yet available as ConnectRPC services.
+
+=== Authentication
+
+If authentication is enabled on your cluster, you must provide credentials with each request, either using HTTP Basic authentication or by including an `Authorization` header with a bearer token. For example:
+
+[,bash]
+----
+curl -u <user>:<password> -X GET "http://localhost:9644/v1/cluster_config"
+----
+
+=== Use legacy (RESTful) endpoints
+
+The base URL for all requests to the legacy endpoints is:
+
+```
+http://<broker-address>:<admin-api-port>/v1/
+```
+
+// TODO: Update link if necessary when v2 URLs are finalized
+For a full list of available endpoints, see the link:/api/doc/admin/[Admin API Reference]. Select "v1" in the version selector to view legacy endpoints.
+
+==== Example request
+
+To use the Admin API to xref:manage:cluster-maintenance/decommission-brokers.adoc[decommission a broker]:
+
+[tabs]
+====
+curl::
++
+--
+Send a PUT request to the link:/api/doc/admin/operation/operation-decommission[`/v1/brokers/\{broker_id}/decommission`] endpoint:
+
+[,bash]
+----
+curl \
+ -u <user>:<password> \
+ --request PUT 'http://<broker-address>:<port>/v1/brokers/<broker-id>/decommission'
+----
+--
+
+rpk::
++
+--
+For Linux deployments only, run xref:reference:rpk/rpk-redpanda/rpk-redpanda-admin-brokers-decommission.adoc[`rpk redpanda admin brokers decommission`]:
+
+[,bash]
+----
+rpk redpanda admin brokers decommission <broker-id>
+----
+--
+====
+
+=== Use new (ConnectRPC) endpoints
```

Comment:
Maybe consider: Use ConnectRPC endpoints

and then put a note explaining what makes them different from v1 endpoints.